packages/react-composites/src/composites/CallComposite/CallControls.tsx

```diff
@@ -1,61 +1,68 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 
-import React, { useCallback } from 'react';
 import {
   CameraButton,
   ControlBar,
   EndCallButton,
   MicrophoneButton,
   OptionsButton,
+  ParticipantMenuItemsCallback,
   ParticipantsButton,
   ScreenShareButton
 } from '@internal/react-components';
-import { groupCallLeaveButtonCompressedStyle, groupCallLeaveButtonStyle } from './styles/CallControls.styles';
+import React, { useCallback } from 'react';
 import { usePropsFor } from './hooks/usePropsFor';
+import { groupCallLeaveButtonCompressedStyle, groupCallLeaveButtonStyle } from './styles/CallControls.styles';
 
 export type CallControlsProps = {
   onEndCallClick(): void;
-  compressedMode?: boolean;
   callInvitationURL?: string;
-  hiddenElements?: CallControlHiddenElements;
+  onFetchParticipantMenuItems?: ParticipantMenuItemsCallback;
+  options?: boolean | CallControlOptions;
 };
 
-export type CallControlHiddenElements = {
+export type CallControlOptions = {
```

Comment:
nit: if we go back to naming it 'Options', we should rename the boolean for buttons with 'hide' to be clear on what they do

Reply:
We can't, since the proposal is to use a shorthand approach. Which means that `CallControlOptions.endCallButton` can have these possible value
`true`
`false`
`{  color: something, position: somthing }` etc etc

Basically, an option can be true, where it will be rendered with default options. Or users can pass in custom options instead of a boolean.